internal/backend/dispatcher/eventsworkflow.go

```diff
@@ -24,6 +25,62 @@ func (d *dispatcher) createEventRecord(ctx context.Context, eventID sdktypes.Eve
 	return nil
 }
 
+func (d *dispatcher) resolveEnv(ctx context.Context, env string) (envID sdktypes.EnvID, err error) {
+	if env == "" {
+		return sdktypes.InvalidEnvID, nil
+	}
+
+	parts := strings.Split(env, "/")
+	switch len(parts) {
+	case 1:
+		if sdktypes.IsEnvID(parts[0]) {
+			return sdktypes.ParseEnvID(parts[0])
+		}
+	case 2:
+		var pid sdktypes.ProjectID
+		if sdktypes.IsProjectID(parts[0]) {
```

Comment:
Same as above - I don't think we should support IDs in URLs.

Reply:
why not?

Reply:
scratch that, you're right.